locale/en/submission.po

```diff
@@ -2073,6 +2073,10 @@ msgstr "View All Submissions"
 msgid "submission.list.viewSubmission"
 msgstr "View Submission"
 
+msgid "submission.list.viewSubmissionSummary"
+msgstr "View Submission Summary"
+
+
```

Comment:
On Decision page need to be able to direct back to the submission summary on new submission listing if the editorial decision is initiated there. Need to show different label to make obvious where user will go.